aries-site/src/layouts/content/DocsPageHeader.js

```diff
@@ -1,20 +1,36 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Box } from 'grommet';
+import Link from 'next/link';
+import { Box, Button, Heading, PageHeader, Text } from 'grommet';
 import { getPageDetails } from '../../utils';
 import { Status, SubsectionText } from '../../components';
-import { Subsection } from '.';
 
 export const DocsPageHeader = ({ title, topic, render }) => {
   const page = getPageDetails(title);
+  const parent = topic && getPageDetails(topic);
 
   return (
-    <Subsection name={render || title} topic={topic} level={1}>
-      <Box>
-        <SubsectionText>{page.description}</SubsectionText>
-        {page.status && <Status status={page.status} />}
-      </Box>
-    </Subsection>
+    <PageHeader
```

Comment:
Overall, I think this played out quite nicely.

Reply:
I agree.